imhotep/repomanagers.py

```diff
@@ -61,6 +61,10 @@ def pull(self, dirname):
 
     def add_remote(self, dirname, name, url):
         log.debug("Adding remote %s url: %s", name, url)
+
+        if url.startswith("https://") and self.authenticated:
```

Comment:
This should probably live above the logging statement on line 63 so folks don't go chasing the wrong URL during debugging.